src/SixLabors.Fonts/RendererOptions.cs

```diff
@@ -82,34 +81,25 @@ public RendererOptions(Font font, float dpiX, float dpiY, Vector2 origin)
         /// <summary>
         /// Gets the font.
         /// </summary>
-        /// <value>
-        /// The font.
-        /// </value>
         public Font Font { get; }
 
         /// <summary>
-        /// Gets or sets the width of the tab.
+        /// Gets or sets the width of the tab. Measured as the distance in spaces.
         /// </summary>
-        /// <value>
-        /// The width of the tab.
-        /// </value>
         public float TabWidth { get; set; } = 4;
 
         /// <summary>
-        /// Gets or sets a value indicating whether [apply kerning].
+        /// Gets or sets a value indicating whether to apply kerning.
```

Comment:
So I'm currently doing this wrong.

There's two types of kerning in an OpenType font. Historical True Type kerning (_kern_) tables and modern glyph positioning (_gpos_) tables.

We're actually turning off glyph positioning and substitution with this switch when we shouldn't be.  

We should probably be allowing GSUB and GPOS always and only toggling kerning in the kern table. The [spec](https://www.w3.org/TR/css-fonts-3/#font-kerning-prop)states the same.

There's a CSS option for turning off [common ligatures](https://developer.mozilla.org/en-US/docs/Web/CSS/font-variant-ligatures) which we should probably adopt also.

Reply:
Hm, I only understand half of this, but if we are not sure the property controls the right thing, maybe better to remove it from the public API for now, and open a tracking issue to figure it out later?

Reply:
Yep. I've done some research and know how this works now. I'll push a fix in a day.

Reply:
Pushed. We now match the CSS specification with a new `KerningMode` property on `RendererOptions`. Unfortunately this will cause runtime errors in ImageSharp.Drawing until we update the `TextOptions` to reflect this (due to copying from ROps to TOps to new ROps instance)